bedrock/firefox/redirects.py

```diff
@@ -24,9 +24,12 @@ def firefox_mobile_faq(request, *args, **kwargs):
     }),
 
     # bug 748503
-    redirect(r'^projects/firefox/[^/]+a[0-9]+/(?:firstrun|whatsnew)(?P<p>.*)$',
+    redirect(r'^projects/firefox/[^/]+a[0-9]+/firstrun(?P<p>.*)$',
```

Comment:
Chatted with @pascalchevrel and agreed to use versioned URLs like /firefox/52.0a1/whatsnew/ for flexibility. The page might literally show What's New in the future :wink: